app-backend/api/src/main/scala/project/ProjectAnnotationRoutes.scala

```diff
@@ -28,52 +28,50 @@ trait ProjectAnnotationRoutes
   implicit val xa: Transactor[IO]
   implicit val ec: ExecutionContext
 
-  def listAnnotations(projectId: UUID): Route = extractTokenHeader { tokenO =>
-    extractMapTokenParam { mapTokenO =>
-      (projectAuthFromMapTokenO(mapTokenO, projectId) |
-        projectAuthFromTokenO(
-          tokenO,
-          projectId,
-          None,
-          ScopedAction(Domain.Projects, Action.Read, None)
-        ) | projectIsPublic(
-        projectId
-      )) {
-        (withPagination & annotationQueryParams) {
-          (page: PageRequest, queryParams: AnnotationQueryParameters) =>
-            complete {
-              (queryParams.withOwnerInfo match {
-                case Some(true) =>
-                  AnnotationDao
-                    .listByLayerWithOwnerInfo(projectId, page, queryParams)
-                    .transact(xa)
-                    .unsafeToFuture
-                    .map { p =>
-                      {
-                        fromPaginatedResponseToGeoJson[
-                          AnnotationWithOwnerInfo,
-                          AnnotationWithOwnerInfo.GeoJSON
-                        ](p)
+  def listAnnotations(projectId: UUID): Route = authenticateAllowAnonymous {
+    user =>
```

Comment:
Example of the general strategy for dealing with "shareable" endpoints - since sharing isn't used for the API any longer I removed that as an option here. I preserved the ability to make something public and accessible though.